app/actions.tsx

```
@@ -277,202 +157,81 @@ export const AI = createAI<AIState, UIState>({
+          case 'reasoning':
+            return {
+              id,
+              component: (
+                <Reasoning isStreaming={false} className="w-full">
+                  <ReasoningTrigger />
+                  <ReasoningContent>{content}</ReasoningContent>
+                </Reasoning>
+              ),
+            };
+          default:
+            const streamableContentDefault = createStreamableValue();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)